mod_reforged/hooks/contracts/contracts/return_item_contract.nut

```diff
@@ -0,0 +1,46 @@
+::mods_hookExactClass("contracts/contracts/return_item_contract", function(o) {
+    local createStates = o.createStates;
+    o.createStates = function()
+    {
+        createStates();
+        foreach (state in this.m.States)
+        {
+            if (state.ID != "Offer") continue;
+            local end = state.end;
+            state.end = function()
+            {
+                end();
+                this.Contract.m.Target.setMovementSpeed(this.Contract.m.Target.getBaseMovementSpeed() * 0.85);  // Thieves are easier to catch
```

Comment:
This seems like an _unnecessary_ freebie for the player. I don't think I've ever had the thieves be impossible to catch. So this isn't a QoL change anymore, but rather a reduction in difficulty. I don't think we need this.

Reply:
I also never had thieves impossible to catch. But there were cases where they start running immediately and it took like 12 hours to catch up.

I believe the fleeing is just part of the vanilla AI. Any party will flee from you if you are stronger than them. So during the early game those thieves will always wait for you patiently. And during the lategame when your party is slower because if its bigger size they start fleeing. At which case catching them is most annoying.

I say, choose one of those mechanics: 
- Either always flee (more realistic) 
- or they never flee (simple mechanic for a simple contract).
I prefer the first one. And I still think there should be some minor movement speed penalty on them. Even if it's just 5%

Reply:
Yeah I agree that it is a bit inconsistent that they don't flee in the early game (when you are weak) and then start fleeing later on. Of the two options you proposed, I agree with your suggestion i.e. **always flee but have a speed penalty of 5%**.

Reply:
I don't mind going for either solution in the future but I don't wanna deal with implementing/fixing that right now in this PR.
I changed the movement speed multiplier to be 0.9, so 10% slower than in vanilla.

Reply:
Fine for now.